src/main/java/net/glowstone/entity/meta/profile/GlowPlayerProfile.java

```diff
@@ -210,11 +211,21 @@ public CompoundTag toNbt() {
         return profileTag;
     }
 
+    @Override
+    public String setName(@Nullable String name) {
+        throw new UnsupportedOperationException("Not implemented yet.");
+    }
+
     @Override
     public UUID getId() {
         return uniqueId == null ? null : uniqueId.getNow(null);
     }
 
+    @Override
+    public UUID setId(@Nullable UUID uuid) {
```

Comment:
I wouldn’t recommend setting the UUID after GameProfile creation. That could lead to unpleasant behavior. 

Player#getGameProfile#setId could be called at any time. Blame me if I’m wrong.

Reply:
The gameprofile is like an identity card. Setting the UUID is comparable to changing the ID of  your IDC after creation.

Reply:
This is a method added upstream by Paper: https://ci.destroystokyo.com/userContent/apidocs/com/destroystokyo/paper/profile/PlayerProfile.html#setId-java.util.UUID-

If I remember correctly, the Paper implementation usually returns copies to avoid this problem. It doesn't matter yet because this is just a stub, and won't be implemented as part of this PR. (I've only added it so the code compiles correctly)